### ISSUE TITLE
Hello!

### ISSUE THREAD

**Issue by cnluminous** [Deltafox79/Navicat_Keygen#22]
Can I translate this software into Chinese and post it on my git website? I hope I can get your authorization? Thank you.

**Comment by Deltafox79** [comment]
Yes of course, it is enough not to remove the credits. Thank you
